samcli/lib/translate/translate_utils.py

```diff
@@ -0,0 +1,31 @@
+"""
+Library for Validating Sam Templates
+"""
+import os
+import logging
+import click
+
+
+LOG = logging.getLogger(__name__)
+
+
+def _read_sam_file(template):
```

Comment:
if it's meant to be used from outside this file, we should remove the leading underscore

Reply:
Removed the underscore